lib/jbuilder.rb

```diff
@@ -281,8 +277,16 @@ def target!
 
   private
 
+  def _extract(object, attributes)
+    if ::Hash === object
+      _extract_hash_values(object, attributes)
+    else
+      _extract_method_values(object, attributes)
+    end
+  end
+
   def _extract_hash_values(object, attributes)
-    attributes.each{ |key| _set_value key, _format_keys(object.fetch(key)) }
+    attributes.each{ |key| _set_value key, _format_keys(object[key]) }
```

Comment:
Also using `[]` instead of `fetch`. It's a bit faster.

```
ruby 3.4.4 (2025-05-14 revision a38531fd3f) +YJIT +PRISM [arm64-darwin24]
Warming up --------------------------------------
                  []     1.582M i/100ms
               fetch     1.627M i/100ms
Calculating -------------------------------------
                  []     26.015M (± 4.2%) i/s   (38.44 ns/i) -    129.730M in   4.999980s
               fetch     21.967M (± 4.2%) i/s   (45.52 ns/i) -    110.641M in   5.047802s

Comparison:
                  []: 26014972.6 i/s
               fetch: 21966990.7 i/s - 1.18x  slower
```

Reply:
Reverted back to `fetch`. What's proposed above results in a change of behaviour: `fetch` throws on missing keys, whereas `[]` returns `nil`.